Networking/Services/SessionManager.swift

```diff
@@ -181,6 +181,7 @@ public extension SessionManager {
         guard isLoggedIn else {
             fatalError("Attempting to update a non logged in session in SessionManager")
         }
+        saveSessionToken(user: user)
```

Comment:
@nahuelwolox 

Update the comment in the protocol for this function. Currently it says: "It will send no notifications, since the session status remains the same."

Reply:
I'm not sure what it does right now because @jcaracciolo made this change. Should I erase ""It will send no notifications, since the session status remains the same." and add something like "it will send a notification if the sessionToken of the user changes"?

Reply:
@nahuelwolox  It should say: "It will send both a session and user notification.", as `login` already says.